client/package.json

```diff
@@ -56,6 +56,8 @@
     "@testing-library/react": "^10.0.3",
     "@testing-library/user-event": "^10.1.0",
     "dotenv": "^8.2.0",
+    "eslint": "^6.8.0",
```

Comment:
isn't this already included in create-react-app? Or does it need to be here if you also add a plugin?

Reply:
Following the other ESLint-related issues, I don't think this should be added in the package.json or it might have conflicting version with the one that is already included and it might cause problems